index.html

```diff
@@ -809,185 +809,197 @@ <h3 class="modal-title" id="warningTitle">
-                      </label>
-                      <input type="text" name="newTalkTitle" placeholder="Talk title (required)" class="form-control required" id="newTalkTitle" required />
-                      <small class="form-text">
-                        This should be the title of the original paper.
-                      </small>
-                    </div>
-                    <div id="talkTitleWarning">
-                      <div class="alert alert-danger">Talk title is required.</div>
-                    </div>
-                  </div>
-                  <div class="col">
-                    <div class="form-group">
-                      <label for="talkNote">
-                        Additional Notes
-                      </label>
-                      <input type="text" name="talkNote" placeholder="Optional: name of presenter, awards, etc" class="form-control" aria-describedby="talkNoteHelp" id="talkNote" maxlength="77" />
-                      <small id="talkNoteHelp" class="form-text">
-                        This is where you might enter additional information
-                        about the paper. Examples include soft merges, awards,
-                        name of presenter, etc. This is not intendeded for
-                        an abstract.
-                      </small>
-                    </div>
-                  </div>
+          <div class="modal-body ps-4 pe-5">
+            <!-- TODO: getting rid of forms if they're not necessary? more reading required -->
+            <form>
+              <input id="talkId" type="hidden" value=""/>
+
+              <!-- TODO: propose we use either a placeholder OR help text but not both (when possible). see https://ux.stackexchange.com/questions/78459/should-help-text-be-above-or-below-the-input
+
+              this is apparently a more lengthy topic than I expected so this is rough notes. let's use placeholder text to suggest 
+              examples or expected format rather than just reiterating instructions (see 
+              https://uxplanet.org/the-anatomy-of-input-field-c3ef863e01d7)'
+              
+              -->
+              <div class="form-group mb-3">
+                <label for="newTalkTitle">
+                  Talk Title
+                </label>
+                <input type="text" name="newTalkTitle" placeholder="Talk title (required)" class="form-control required my-1" id="newTalkTitle" required />
+                <small class="form-text">
+                  This should be the title of the original paper.
+                </small>
+              </div>
+              <div id="talkTitleWarning">
+                <div class="alert alert-danger">Talk title is required.</div>
+              </div>
+
+              <div class="form-group mb-3">
+                <label for="newTalkAuthors">
+                  Talk Author(s)
+                </label>
+                <!-- TODO: shouldn't talk authors be a required field?' -->
```

Comment:
Ordinarily I would agree that it makes sense to include authors, but there are circumstances where it might be omitted in a rump session.

Reply:
removing required (both classes and attributes)